tronpy/providers/async_http.py

```diff
@@ -22,6 +23,7 @@ def __init__(
         timeout: float = DEFAULT_TIMEOUT,
         client: httpx.AsyncClient = None,
         api_key: str = DEFAULT_API_KEY,
+        jw_token: str = None,
```

Comment:
Doesn't async client need same kind of validation like in sync one? Also check the comment below

Reply:
okay, i will include it.